app/assets/stylesheets/modules/airline_show.scss

```diff
@@ -1,3 +1,15 @@
+@import 'variables';
+
 .date {
   text-align: right;
 }
+
+.postfix.button {
+  padding-left: 0;
```

Comment:
Properties should be ordered background-color, border, padding-bottom, padding-left, padding-right, padding-top, text-align